### PR TITLE
P#62757 Disable duplicate check and add warning

### PR DESCRIPTION
**Pull request by hungnc89**

### CODE REVIEW

File: plugin/Gui/AdminPageFormSettingsContact.php [hunk 1]
```diff
@@ -138,11 +138,13 @@ protected function buildForms()
 
 		if ($this->_showCheckDuplicates) {
 			$pInputModel = $pInputModelBuilder->build(InputModelDBFactoryConfigForm::INPUT_FORM_CHECKDUPLICATES);
+			$pInputModel->setHint(__('Be aware that when activated the duplicate checkcan overwrite address records. This function will be removed in the future. Use at your own risk.', 'onoffice-for-wp-websites'));
```
Comment (jayay) [inline plugin/Gui/AdminPageFormSettingsContact.php:141]: Typo: "check can"

**Review by jayay — COMMENTED** [review]
After the deactivation of the duplicate check, the affected forms should be listed to the user.  It'd be nice if you could include that

File: plugin/Controller/AdminViewController.php [hunk 2]
```diff
@@ -390,6 +396,24 @@ public function displayAPIError()
...
+
+	/**
+	 *
+	 */
+
+	public function displayDeactivateDuplicateCheckWarning()
+	{
+		if (get_option('onoffice-duplicate-check-warning', 'onoffice-for-wp-websites') === "1") {
+			$class = 'notice notice-error duplicate-check-notify is-dismissible';
+			/* translators: %s will be replaced with the translation of 'API token and secret'. */
```
Comment (jayay) [inline plugin/Controller/AdminViewController.php:408]: `/* translators:` comment is not necessary here

**Review by jayay — COMMENTED** [review]
Looking good so far. Could you please add more tests?

**Review by jayay — APPROVED** [review]

**Comment by michaelarnold-dev** [comment]
@hungnc89 Could you please improve the code coverage a bit?

**Comment by hungnc89** [comment]
@michaelarnold-dev Here you are.

**Pull request merged**